### ISSUE TITLE
Pdf view blocked when we open more than one at the same time

### ISSUE THREAD

**Issue by Gabrielle-anne** [stephanrauh/ngx-extended-pdf-viewer#2396]
Discussed in https://github.com/stephanrauh/ngx-extended-pdf-viewer/discussions/2335

Hello, 
I opened an issue because the discussion was somehow labeled as solved, 
But it still doesn’t work, and I don’t know how to put it back in unsolved.
I can’t change the pages on the second pdf without reloading, but on the first one all is working perfectly.
The bug appears when I do this :
```ts
openPdf() {
window.open('/pdf', '_blank')
window.open('/pdf', '_blank')
window.open('/pdf', '_blank')
}
```
I tried this : [test 4.zip](https://github.com/user-attachments/files/15911292/test.4.zip)
Is there something I did wrong ?

Thank you very much, for your help.

**Comment by stephanrauh** [comment]
Yes, I remember your message, and it's still in my inbox. Sorry about answering sluggishly - currently I'm drowing in a flood of tickets, and some of them are very convoluted.

I've marked your discussion as solved because I'd managed to reproduce the issue, and after that, I'd managed to solve that. Now I wonder if there's a difference between your project and my project. Or if there's something else going on.

**Comment by Gabrielle-anne** [comment]
Hello, 
I've managed in zip I sent in my message, to reproduce the problem each time no matter what.
Are you sure you can move as you please on the second PDF without reloading ?  I use Angular 15.1 and the version of the library I use is the 18.1.10
Thank you for your time,

**Comment by stephanrauh** [comment]
I've downloaded your test.4.zip and got it up and running. I can navigate between the pages without problems, no matter how many tabs I open or which one I try. The problem was about page navigation, right?

**Comment by Gabrielle-anne** [comment]
Hello, 
I am very surprised that you didn't reproduce this bug, I tried on 2 computers and on other navigators and it's always showing up. 
So I desperately searched on my own, and then I found this :
![image](https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/71768679/bd1acff6-ec65-4175-a155-3b18c6018485)

When the bug shows up : the location is filled (location.pageNumber =1 and we can't navigate on the pdf). 
I think it's because the program go systematically back on the first page. When it's working fine, the location is null.

**Comment by stephanrauh** [comment]
That's remarkably strange. What is the location, and why does it interfere here? Maybe it's a timing problem. The location is set to the first visible page here, and maybe this method is called too early and that's why why navigation is broken:
<img width="982" alt="image" src="https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/3045767/f251f0aa-795f-4f9d-82db-4e3896e78c6b">

**Comment by stephanrauh** [comment]
Can you test your application with version 21.1.0 again, please?

**Comment by stephanrauh** [comment]
I haven't heard from you for a long time. I'm closing the ticket now.
